static/app/views/explore/components/schemaHints/schemaHintsList.tsx

```diff
@@ -83,12 +83,30 @@ function getTagsFromKeys(keys: string[], tags: TagCollection): Tag[] {
 export function addFilterToQuery(
   filterQuery: MutableSearch,
   tag: Tag,
-  isBoolean: boolean
+  fieldDefinition: FieldDefinition | null
 ) {
-  filterQuery.addFilterValue(
-    tag.key,
-    isBoolean ? 'True' : tag.kind === FieldKind.MEASUREMENT ? '>0' : ''
-  );
+  if (tag.kind === FieldKind.FUNCTION) {
+    const defaultFunctionParam = fieldDefinition?.parameters?.[0]?.defaultValue;
+    filterQuery.addFilterValue(
+      defaultFunctionParam ? `${tag.key}(${defaultFunctionParam})` : `${tag.key}()`,
+      '>0'
+    );
```

Comment:
nit: 
```suggestion
    const defaultFunctionParam = fieldDefinition?.parameters?.[0]?.defaultValue ?? '';
    filterQuery.addFilterValue(`${tag.key}(${defaultFunctionParam})`, '>0');
```
feels like better readability